hack/dockerfile/binaries-commits

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
 TOMLV_COMMIT=9baf8a8a9f2ed20a8e54160840c492f937eeaf9a
-RUNC_COMMIT=2f7393a47307a16f8cee44a37b262e8b81021e3e
+
+# When updating RUNC_COMMIT, also update libcontainerd in vendor.conf accordingly
```

Comment:
ah, was doubting, because that line especially refers to libcontainer; will update, hold on